### ISSUE TITLE
Bug: After playing Golf 1.7-1.8 on Win64, browser screen goes whiter on newly opened window

### ISSUE THREAD

**Issue by dosxga** [fallahn/crogine#75]
After playing Golf 1.8 and exit on Win64, browser screen thats newly opened browser windows goes whiter with hard to read text.  have to close game and log out and in.  Will try to sort out or post screenshot.  **but the fix is to quit game and log out and back in.**  Don't have access to latest version yet.  But probably won't matter.  Using chrome based browser.

**Comment by fallahn** [comment]
I'm not sure I follow - you're saying that playing golf affects the rendering of your browser? I don't really understand how... golf, even the very old versions, has never used any sort of browser integration.

What platform is this, and which GPU do you have (Intel integrated GPUs are known to be able to cause problems)? The only thing I can think of is that it's some sort of graphics driver problem.

**Comment by dosxga** [comment]
or an audio driver, maybe
Processor	Intel(R) Core(TM) i9-10900KF CPU @ 3.70GHz   3.70 GHz, driver version 531.61
nvidia 3070 I believe
![2023-05-01 14_23_44 - NahimicSvc32 exe Application _ Tom's Hardware Forum - Brave](https://user-images.githubusercontent.com/54957125/235515575-45af0084-a172-4bd3-ab2e-f0b4e015089e.png)

wouldn't know about how to error trap and debug.  but the fix should help others.
 enclosed is just an example fullscreen of tom's hardware....
maybe its an error with HDR

**Comment by fallahn** [comment]
Wow that's weird - I've never heard of anything like it :S Does it still happen with version 1.9 https://github.com/fallahn/crogine/releases/tag/golf-v1.9.1 ?

**Comment by dosxga** [comment]
yes.  I should have mentioned I cancel the Windows Firewall prompt and it doesn't show any news.  Updated title in bug report.

**Comment by fallahn** [comment]
OK, thanks. The news makes sense because the RSS URL changed and the game doesn't know how to follow redirects. Still no idea why it affects your browser though. 🤔
